packages/OrderedList/OrderedList.jsx

```diff
@@ -1,35 +1,40 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
 import { componentWithName } from '@tds/util-prop-types'
 
 import Box from '@tds/core-box'
-import typographyStyle from '@tds/shared-typography/Typography.modules.scss'
+import { list } from '@tds/shared-styles'
 
 import safeRest from '../../shared/utils/safeRest'
-import joinClassNames from '../../shared/utils/joinClassNames'
 
-import OrderedItem from './OrderedItem/OrderedItem'
+import OrderedItem, { StyledOrderedItem } from './OrderedItem/OrderedItem'
 
-import styles from './OrderedList.modules.scss'
+const listStyleType = {
+  upperAlpha: 'upper-alpha',
+  lowerAlpha: 'lower-alpha',
+  decimal: 'decimal',
+}
 
-const injectListStyle = (child, size) => React.cloneElement(child, { size })
+export const StyledOrderedList = styled(({ size, listStyle, ...rest }) => <Box {...rest} />)(
+  ({ listStyle }) => ({
+    paddingLeft: '3rem',
+    listStyleType: listStyleType[listStyle],
+    ...list.nestedListSpacing,
```

Comment:
Please double check the implementation of this.

![image](https://user-images.githubusercontent.com/12798751/57238314-1a475b00-6ff7-11e9-9b92-2fe0f61de85f.png)